doc/nrf/known_issues.rst

```diff
@@ -277,45 +285,110 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-5-1
 
-KRKNWK-7836: Coordinator asserting when flooded with ZDO commands
-  Executing a high number of ZDO commands can cause assert on the coordinator with the :ref:`lib_zigbee_shell` component enabled.
+KRKNWK-8478: NCP host application crash on exceeding :c:macro:`TX_BUFFERS_POOL_SIZE`
+  If the NCP host application exceeds the :c:macro:`TX_BUFFERS_POOL_SIZE` pending requests, the application will crash on an assertion.
 
-.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
+  **Workaround**: Increase the value of :c:macro:`TX_BUFFERS_POOL_SIZE` or define shorter polling interval (:c:macro:`NCP_TRANSPORT_REFRESH_TIME`).
+
+.. rst-class:: v1-5-1
+
+KRKNWK-8200: Sleepy End Device halts during the commissioning
+  If the turbo poll is disabled in the ``ZB_BDB_SIGNAL_DEVICE_FIRST_START`` signal, SED halts during the commissioning.
+
+  **Workaround**: Use the development libraries link or use ``ZB_BDB_SIGNAL_STEERING`` signal with successful status to disable turbo poll.
+  See the following snippet for an example:
+
+  .. code-block:: c
+
+     /* Workaround for KRKNWK-8200 (turbo poll) */
+     switch(sig)
+     {
+     case ZB_BDB_SIGNAL_DEVICE_REBOOT:
+     case ZB_BDB_SIGNAL_STEERING:
+             if (status == RET_OK) {
+                     zb_zdo_pim_permit_turbo_poll(0);
+                     zb_zdo_pim_set_long_poll_interval(2000);
+             }
+             break;
+     }
+
+.. rst-class:: v1-5-1
+
+KRKNWK-8200: Successful signal on commissioning fail
+  A successful steering signal is generated if the commissioning fails during TCLK exchange.
+
+  **Workaround**: Use the development libraries link or check for Extended PAN ID in the steering signal handler.
+  If it is equal to zero, handle the signal as if it had unsuccessful status.
+  See the following snippet for an example:
+
+  .. code-block:: c
+
+     /* Workaround for KRKNWK-8200 (signal status) */
+     switch(sig)
+     {
+     case ZB_BDB_SIGNAL_STEERING:
+             if (status == RET_OK) {
+                     zb_ext_pan_id_t extended_pan_id;
+                     zb_get_extended_pan_id(extended_pan_id);
+                     if (!(ZB_IEEE_ADDR_IS_VALID(extended_pan_id))) {
+                            zb_buf_set_status(bufid, -1);
+                            status = -1;
+                     }
+             }
+             break;
+     }
+
+.. rst-class:: v1-5-1
+
+KRKNWK-9461 / KRKNWK-9596 : Multiprotocol samples crashes with some smartphones
```

Comment:
This is also applicable for Thread multiprotocol samples.

Reply:
Added the issue in the Thread section also.